test/unittest/dds/xtypes/type_conversion/types/idls/alias_struct.idl

```diff
@@ -0,0 +1,12 @@
+typedef short MyShort;
+
+typedef unsigned long MyLong;
+
+typedef MyShort MyRecursiveShort;
+
+struct alias_struct
+{
+    MyShort my_short;
+    MyLong my_long;
+    MyRecursiveShort my_recursive_short;
+};
```

Comment:
We have a common repository containing the IDL files that are testes in all products ... or will be. Fast-DDS already has this repository in `thirdparty/dds-types-test`. I suggest to use these IDL files instead of new ones.

Reply:
That'd be ideal, yes. The problem is that the DynamicType to IDL serializer serializes a single DynamicType to IDL. The IDLs in `thirdparty/dds-types-test`, however, contain many different types. Furthermore, the DynamicType to IDL serializer can omit information present in the IDL but not present in the DynamicType. For example, a DynamicType has no way of knowing if value was explicitly set by the user to its default value, or if the user didn't set it. Any ideas on how to fix these issues?

Reply:
Maybe in the future, when we have the IDL runtime parser we can test DynamicType -> IDL -> DynamicType and compare the original DynamicType and the result.